### PR TITLE
handle eventual consistency

### PR DESCRIPTION
**Pull request by yanns**
@schleichardt please review

### CODE REVIEW

File: sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java [hunk 1]
```diff
@@ -15,6 +15,5 @@ public void execution() throws Exception {
         assertThat(project.getCountries()).overridingErrorMessage("countries").isNotEmpty();
         assertThat(project.getLanguages()).overridingErrorMessage("languages").isNotEmpty();
         assertThat(project.getCreatedAt()).overridingErrorMessage("createdAt").isNotNull();
-        assertThat(project.getTrialUntil()).overridingErrorMessage("trialUntil").isNotNull();
```
Comment (schleichardt) [inline sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java]: Why do you need to remove this?

Comment (yanns) [inline sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java]: because the project used on CI does not have this field (it is optional).
Or we could change the project on CI.

Comment (yanns) [inline sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java]: I changed the project in TC and will revert this change.

Comment (yanns) [inline sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java]: done

**Comment by schleichardt** [comment]
ready for merge if travis is green

**Comment by schleichardt** [comment]
Thanks! That will help me much in the future.
